### PR TITLE
Use channel name and channel id.

### PR DESCRIPTION
**Pull request by samgiles**

### CODE REVIEW

**Comment by samgiles** [comment]
r? @wilsonpage 

Once this is merged I can update the channel names on the "production" server.

File: content-service/migrations/20161129113620-channel-ids.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      resolve(data);
+    });
+  })
+  .then(function(data) {
+    return db.runSql(data);
+  });
+};
+
+exports._meta = {
+  "version": 1
```
Comment (wilsonpage) [inline content-service/migrations/20161129113620-channel-ids.js:52]: NIT: Quotes?

File: content-service/server/database/beacons.js [hunk 2]
```diff
@@ -97,13 +97,15 @@ module.exports = function(knex) {
       });
   }
 
-  function getBeaconInfo(slug, channelName) {
+  function getBeaconInfo(slug, channelId) {
+    console.log("SHORT ID: ", slug);
+    console.log("CHANNEL ID: ", channelId);
```
Comment (wilsonpage) [inline content-service/server/database/beacons.js:102]: You wanna keep these logs?

File: content-service/server/database/beacons.js [hunk 3]
```diff
@@ -97,13 +97,15 @@ module.exports = function(knex) {
...
     const constraints = {
-      id: shortIdToNum(slug),
+      'beacon.id': shortIdToNum(slug),
```
Comment (wilsonpage) [inline content-service/server/database/beacons.js:104]: Consistency: Is there a reason why we have `channel_id` and `beacon.id`?

File: content-service/server/express/errorhandler.js [hunk 4]
```diff
@@ -30,5 +30,6 @@ module.exports = function(err, req, res, next) {
   }
 
   res.writeHead(statusCode, headers);
+  console.log(msg);
```
Comment (wilsonpage) [inline content-service/server/express/errorhandler.js]: Shoudl this be removed?

File: content-service/migrations/20161129113620-channel-ids.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+var dbm;
+var type;
+var seed;
+var fs = require('fs');
+var path = require('path');
+var Promise;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
```
Comment (wilsonpage) [inline content-service/migrations/20161129113620-channel-ids.js:12]: This comment doesn't mean much to me. I dunno what `dbmigrate` is or `dependency on NODE_PATH` means :) But perhaps I'm not the intended reader.

File: content-service/migrations/20161129113620-channel-ids.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
+'use strict';
```
Comment (wilsonpage) [inline content-service/migrations/20161129113620-channel-ids.js:1]: Would be nice to have a high-level explainer on how these migrations work.

File: content-service/migrations/20161129113620-channel-ids.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+      resolve(data);
+    });
+  })
+  .then(function(data) {
+    return db.runSql(data);
+  });
+};
+
+exports.down = function(db) {
```
Comment (wilsonpage) [inline content-service/migrations/20161129113620-channel-ids.js:36]: When would we migrate down a version?

File: content-service/migrations/20161129113620-channel-ids.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+  var filePath = path.join(__dirname, 'sqls', '20161129113620-channel-ids-down.sql');
+  return new Promise( function( resolve, reject ) {
```
Comment (wilsonpage) [inline content-service/migrations/20161129113620-channel-ids.js:38]: Consistency: Strange spacing between arguments.

**Review by wilsonpage — APPROVED** [review]
Some questions, documentation and nits, otherwise seems simple enough.

r+

**Comment by samgiles** [comment]
I'm going to close this, as it's likely not applicable with new direction.  We can keep the branch around for now though just in case.

**Pull request closed without merging**